include/libvfio-user.h

```diff
@@ -316,24 +313,28 @@ vfu_setup_device_reset_cb(vfu_ctx_t *vfu_ctx, vfu_reset_cb_t *reset);
  * Function that is called when the guest maps a DMA region. Optional.
  *
  * @vfu_ctx: the libvfio-user context
- * @iova: iova address
- * @len: length
+ * @iova: IOVA address and length
```

Comment:
maybe expand what this is?

Reply:
Something like the followig?

"I/O virtual address and length. This is the guest physical address the guest registers for DMA. This address range refers to the guest's memory and is invalid in the user's address space."

Reply:
"physical address range" but yeah, thanks